### PR TITLE
Add EventSource polyfill

### LINKED ISSUE #996: Feature request: Add EventSource polyfill
Either https://github.com/Yaffle/EventSource or https://github.com/amvtek/EventSource.
It's needed in old Androids and all IEs/Edge.

### PR DESCRIPTION
**Pull request by CheloXL**
Closes #996

This pull requests adds the EventSource polyfill to browsers that doesn't have it. I Added several browsers to the list based on data from caniuse.

I didn't add any tests, as I'm not sure how/if I can write some since that would requiere some server side code.

Please let me know if there is anything else that I would need to do in order to make this PR a good one.

### CODE REVIEW

**Comment by triblondon** [comment]
✔️  Tests present on the upstream repo
✔️  Upstream repo is well followed, released recently

We'd prefer the Android Semver constraint to be `<`, not `<=`, so if evensource was added in 4.4, make it `<4.4`.  Otherwise this looks good to me.

**Comment by CheloXL** [comment]
@triblondon Done!

**Comment by CheloXL** [comment]
@triblondon Do you have an ETA on when this may be merged? Thanks!

**Review by JakeChampion — APPROVED** [review]

**Review by rowanmanning — APPROVED** [review]

File: polyfills/EventSource/config.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+	"aliases": [
+	],
+	"browsers": {
+		"firefox": "<36",
+		"chrome": "<26",
+		"safari": "<7",
+		"opera": "<12",
+
```
Comment (leggsimon) [inline polyfills/EventSource/config.json]: Remove empty line?

**Pull request merged**